src/pages/Home/WelcomePage.js

```diff
@@ -29,7 +31,10 @@ const WelcomePage = () => {
           onChange={(e) => setUserInput(e.target.value)}
         />
         {userInput && (
```

Comment:
```suggestion
        {!!userInput && (
```
This is something we should look out for when adding checks inside render methods. If we don't want render anything unnecessary we should always use double negation to convert non boolean values into booleans

Reply:
changed it. 
03a15382890102e8154d0750303fda3b7254c4cf